kolibri/core/assets/src/styles/core-global.styl

```diff
@@ -99,6 +99,11 @@ button
   &:focus
     outline: $core-action-light 2px solid
 
+  &:disabled
+    color: $core-text-annotation
+    border-color: $core-text-annotation
+    cursor: default
+
```

Comment:
This has nothing to do with the present PR, but after messing with styles for a while, I wanted to roll back on a commit, and decided to grab the core global styles from the latest master.